### PR TITLE
Updated Quick and Nimble (Swift 4.2+ support)

### PR DESCRIPTION
**Pull request by jakovi89**
Updated Quick and Nimble for swift 4.2 support

### CODE REVIEW

File: ReactiveObjC.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2823,7 +2824,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				ENABLE_TESTABILITY = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```
Comment (mdiep) [inline ReactiveObjC.xcodeproj/project.pbxproj]: Can you change the deployment target of the test target only? The actual framework should be able to stay deployable to 10.9.

**Comment by jakovi89** [comment]
Thanks a lot @mdiep, I'll do another PR for the ObjcBridge in a moment

**Comment by mdiep** [comment]
Looks like some of the tests still need a macOS 10.10 deployment target.

**Comment by jakovi89** [comment]
You're right - Is this set somewhere else than in the project file?

**Comment by mdiep** [comment]
Thanks!

**Pull request merged**